spec/unleash/scheduled_executor_spec.rb

```diff
@@ -55,39 +55,39 @@
     max_exceptions = 1
 
     scheduled_executor = Unleash::ScheduledExecutor.new('TesterLoop', 0.02, max_exceptions, true)
-    new_instance_id = SecureRandom.uuid
-    original_instance_id = Unleash.configuration.instance_id
+    new_app_name = SecureRandom.uuid
+    original_app_name = Unleash.configuration.app_name
 
     scheduled_executor.run do
-      Unleash.configuration.instance_id = new_instance_id
+      Unleash.configuration.app_name = new_app_name
       raise StopIteration
     end
 
     sleep 0.01
 
-    expect(Unleash.configuration.instance_id).to eq(new_instance_id)
+    expect(Unleash.configuration.app_name).to eq(new_app_name)
     scheduled_executor.thread.join
 
-    Unleash.configuration.instance_id = original_instance_id
+    Unleash.configuration.app_name = original_app_name
   end
 
   xit "will not trigger immediate exection when not set" do
     max_exceptions = 1
 
     scheduled_executor = Unleash::ScheduledExecutor.new('TesterLoop', 0.02, max_exceptions, false)
-    new_instance_id = SecureRandom.uuid
-    original_instance_id = Unleash.configuration.instance_id
+    new_app_name = SecureRandom.uuid
+    original_app_name = Unleash.configuration.app_name
 
     scheduled_executor.run do
-      Unleash.configuration.instance_id = new_instance_id
+      Unleash.configuration.app_name = new_app_name
       raise StopIteration
     end
 
     sleep 0.01
 
-    expect(Unleash.configuration.instance_id).to eq(original_instance_id)
+    expect(Unleash.configuration.app_name).to eq(original_app_name)
     scheduled_executor.thread.join
 
-    Unleash.configuration.instance_id = original_instance_id
+    Unleash.configuration.app_name = original_app_name
```

Comment:
I don't think the property here really mattered (this should work with any property) so I picked `app_name` instead.